script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -158,21 +158,21 @@ Then that means I can kill even someone like you. I can kill anything, because I
 Arcueid turns off the lights in the room and moves to the window.
 The curtains are opened.
 The previously hidden profusion of lights and details come into view.
-The moonlight streaming through the window casts a shadow of a white woman.
+The moonlight streaming through the window casts a shadow of a pale figure.
 <Page46>
 "Come on, it's fine so don't hold back.
-Oh, could those glasses be preventing you from seeing it by any chance?"
+Oh, could those glasses be preventing you from seeing them by any chance?"
 <Page47>
 "―――It's okay, right?"
 ...Impossible.
-I put my hand on my glasses, which I've never taken off in front of someone before.
+I bring my hand up to touch my glasses, which I've never taken off before in front of someone else before.
```

Comment:
the added before should be removed

Reply:
I bring my hand up to touch my glasses, something I've never done in front of anyone else before.